postgres-pubsub.test.js

```diff
@@ -172,7 +172,7 @@ describe("PostgresPubSub", () => {
       expect(result).not.toBeUndefined();
```

Comment:
❗ Apparently this expectation is failing but was flying under the radar. Adding the `.catch(done)` on L105 caused it to be visible:
```
  ● PostgresPubSub › AsyncIterator should not trigger event on asyncIterator already returned

    expect(received).not.toBeUndefined()

    Received: undefined

      171 |     iterator.next().then(result => {
      172 |       expect(result).not.toBeUndefined();
    > 173 |       expect(result.value).not.toBeUndefined();
          |                                ^
      174 |       expect(result.done).toBe(false);
      175 |     }).catch(done);
      176 |

      at postgres-pubsub.test.js:173:32
```